src/components/views/messages/ReactionsRowButton.tsx

```diff
@@ -130,9 +131,18 @@ export default class ReactionsRowButton extends React.PureComponent<IProps, ISta
                 onMouseOver={this.onMouseOver}
                 onMouseLeave={this.onMouseLeave}
             >
-                <span className="mx_ReactionsRowButton_content" aria-hidden="true">
-                    {content}
-                </span>
+                {content.startsWith("mxc://") ? (
```

Comment:
Does matrix-react/js-sdk not have a more generic MXC URI validation function already?

Reply:
It seems like this method is used across the codebase: https://github.com/search?q=repo%3Amatrix-org%2Fmatrix-react-sdk+%22.startsWith%28%5C%22mxc%3A%2F%2F%5C%22%29%22&type=code.

I couldn't find a helper function for this.